OpenRA.Mods.Common/Commands/DevCommands.cs

```diff
@@ -85,10 +87,29 @@ public void InvokeCommand(string name, string arg)
 					IssueDevCommand(world, "DevEnableTech");
 					IssueDevCommand(world, "DevFastCharge");
 					IssueDevCommand(world, "DevGiveCash");
+					IssueDevCommand(world, "DevLevelUp");
 					break;
 
 				case "crash":
 					throw new DevException();
+				
+				case "levelup":
+					var level = 0;
```

Comment:
You can default to `level = 1` and then remove the `else GiveLevels(1);` in GainsExperience.